Maple2.Server.DebugGame/Graphics/Scene/Camera.cs

```diff
@@ -5,19 +5,55 @@ namespace Maple2.Server.DebugGame.Graphics.Scene;
 
 public class Camera {
     public Transform Transform { get; init; } = new Transform();
-    public Matrix4x4 ProjectionMatrix { get; private set; }
+    public Matrix4x4 ProjectionMatrix { get; private set; } = Matrix4x4.Identity;
+    public Matrix4x4 ViewMatrix { get; private set; } = Matrix4x4.Identity;
```

Comment:
This is what the Transform member is for. This whole variable should be removed & references rerouted to Transform. If you need to make it point towards something then do that in the camera controller.